docs/configuration.rst

```diff
@@ -266,6 +272,37 @@ That'll pull the ``DATABASE_URL`` value from the environment (it throws an error
 if it's not there) and runs it through ``dj_database_url`` which parses it and
 returns what Django needs.
 
+.. Note::
+
+   To use dj-database-url, you'll need to install it separately. Everett doesn't
+   require it to be installed.
+
+
+django-cache-url
+----------------
+
+Everett works great with `django-cache-url <https://github.com/ghickman/django-cache-url>`_.
+
+For example::
+
+    import django_cache_url
+    from everett.manager import ConfigManager, ConfigOSEnv
+
+    config = ConfigManager([ConfigOSEnv()])
+    CACHES = {
+        'default': config('CACHE_URL', parser=django_cache_url.parse)
```

Comment:
I think I'd throw a `default` on either this one or the `dj_database_url` one just to demonstrate a slightly more interesting default. For the database one e.g. you could use `config('DATABASE_URL', parser=dj_database_url.parse, default='sqlite:///my.db')`.